cypress/e2e/companyShowSpec.cy.js

```diff
@@ -0,0 +1,141 @@
+import { mockCompanies } from "../fixtures/mockCompanies.js";
```

Comment:
Great work, but remember that you can also just do the login process like in the other cy tests.